src/api/settings/getServerSettings.js

```diff
@@ -35,6 +35,10 @@ export type ApiResponseServerSettings = {|
   realm_uri: string,
   require_email_format_usernames: boolean,
   zulip_version: string,
+  // zulip_feature_level added for server v2.2, feature level 1
+  // See https://github.com/zulip/zulip/commit/e3b90a5ec8 for
+  // the server-side implementation.
```

Comment:
Instead of the implementation, we can link to the docs:
  https://zulipchat.com/api/server-settings

Implementation works as a substitute when there are no docs, but docs are better when available.